### DISCUSSION TITLE
ESMValCore feature freeze v2.5.0

### DISCUSSION THREAD

**Discussion by axel-lauer** [ESMValGroup/ESMValCore#1450] [Announcements]
@ESMValGroup/esmvaltool-developmentteam 

The next release of ESMValCore (and ESMValTool) is coming up very soon! The ESMValCore feature freeze for v2.5.0 is scheduled for next Monday (**7 February 2022**). Please let us know **as soon as possible** if you are still working on something for this release by adding your issue or pull request to our v2.5.0 milestone: https://github.com/ESMValGroup/ESMValCore/milestone/9

**Comment by remi-kazeroni** [comment]
An issue on the recipe testing for the Core release is now available: https://github.com/ESMValGroup/ESMValTool/issues/2535
